object_manager.go

```diff
@@ -157,7 +160,7 @@ func BuildNetworkFromRef(ref string) *Network {
 	}
 }
 
-func (objMgr *ObjectManager) GetNetwork(netview string, cidr string, ea EA) (*Network, error) {
+func (objMgr *ObjectManager) GetNetwork(netview string, cidr string, eaSearch EASearch) (*Network, error) {
```

Comment:
You shouldn't change the signature.

More importantly, EAS is meant for internal use only. As far as the user is concerned, it is just EA.

Since EAS and EA are basically the same beast, you can just do a type conversion inside the function when you assign that to `network.eaSearch`
i.e.:
`network.eaSearch = EAS(ea)`